.azure-pipelines/azure-pipelines-win.yml

```diff
@@ -10,10 +10,10 @@ jobs:
   strategy:
     maxParallel: 4
     matrix:
-      win_c_compilervs2008cxx_compilervs2008python2.7:
-        CONFIG: win_c_compilervs2008cxx_compilervs2008python2.7
-        CONDA_BLD_PATH: D:\\bld\\
-        UPLOAD_PACKAGES: True
+      # win_c_compilervs2008cxx_compilervs2008python2.7:
+      #   CONFIG: win_c_compilervs2008cxx_compilervs2008python2.7
+      #   CONDA_BLD_PATH: D:\\bld\\
+      #   UPLOAD_PACKAGES: True
```

Comment:
We don't edit the CI config directly. Let me add the skip statement here for you.